src/DssSpell.t.sol

```diff
@@ -2232,6 +2232,33 @@ contract DssSpellTest is DSTest, DSMath {
         checkAuth(true);
     }
 
+    // SES auditors Multisig
+    address constant SES_AUDITORS_MULTISIG = 0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6;
+    // Monthly expenses
+    uint256 constant SES_AUDITORS_AMOUNT = 1_153_480;
+
+    function test_pe_core_unit_budgets() public {
```

Comment:
I think the function could be more generally renamed to `test_core_unit_budgets`

Reply:
Done